CONTRIBUTING.md

```diff
@@ -144,12 +144,13 @@ The title of your pull request should
 
 - briefly describe and reflect the changes
 - wrap any code with backticks
+- no period in the PR title
```

Comment:
A detail but to match the grammatical style. It should naturally follow "The title of your pull request should
".
```suggestion
- not end with a period
```